apps/obsidian/src/utils/generateUid.ts

```diff
@@ -1,4 +1,4 @@
-import { nanoid } from "nanoid";
+import nanoid from "nanoid";
```

Comment:
Interesting, I wonder why this diff is showing up.  Looks like it actually is 
`import nanoid from "nanoid";` in `main` (in my IDE and GitHub)

<img width="1224" height="809" alt="image" src="https://github.com/user-attachments/assets/da5968e2-35f8-4760-866b-51de09bbd987" />